app/javascript/components/RecordingMethodsPreviews/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import PreviewBase from '../Shared/PreviewBase'
+import WhichWeWillRecordUsing from '../Shared/WhichWeWillRecordUsing'
+import WhichWeWillRecordUsingSentence from '../Shared/WhichWeWillRecordUsingSentence'
+
+class RecordingMethodsPreviews extends PreviewBase {
+  componentName () {
+    return 'RecordingMethodsPreviews'
+  }
+
+  render () {
+    return (
+      <div className="reactive-container__preview">
+        <section className='reactive-preview__section'>
+          <h3 className="subtitle-small" id="what">What happens in this research session?</h3>
+          <p>&hellip;</p>
+          <WhichWeWillRecordUsing {...this.state } />
```

Comment:
If we have `WhichWeWillRecordUsingSentence`, I think this should be called `WhichWeWillRecordUsingList`